docs/how-to/monitor-your-backups.md

```diff
@@ -562,6 +563,58 @@ Heartbeat Retry = 360          # = 10 minutes
 Resend Notification every X times = 1
 ```
 
+## zabbix hook
+
+<span class="minilink minilink-addedin">New in version 1.8.15</span>
+[zabbix](https://www.zabbix.com/) is an open-source monitoring tool used for tracking and managing the performance and availability of networks, servers, and applications in real-time.
+
+This hook does not do any notifications on it's own. Instead, it relies on
+your zabbix instance to notify and perform escalations based on the zabbix
+configuration. The `states` list will choose which states to trigger on. 
+Each state can have its own custom values configured. These values are 
+populated in the item data in zabbix. If none are provided, will use the
+default.
+
+An example configuration is shown here with all the available options.
+
+```yaml
+zabbix:
+    server: http://cloud.zabbix.com/zabbix/api_jsonrpc.php
+    
+    username: myuser
+    password: secret
+    api_key: b2ecba64d8beb47fc161ae48b164cfd7104a79e8e48e6074ef5b141d8a0aeeca
+
+    host: "borg-server"
+    key: borg.status
+    itemid: 55105
+
+    start:
+        value: "STARTED"
+    finish:
+        value: "OK"
+    fail:
+        value: "ERROR"
+    states:
+        - start
+        - finish
+        - fail
+```
+
+<span class="minilink minilink-addedin">Zabbix 7.0+</span>
```

Comment:
I think the convention for these sub-headers would actually be more like: `###  Zabbix 7.0+`. Although maybe you're wanting them to be inline with the following paragraph?

Reply:
I was doing my best to try to follow the standard but I am happy to change it 😃 https://github.com/borgmatic-collective/borgmatic/pull/85/commits/c1a08edca23a4f60512eeabfb1a1274ff6a78ea1